### ISSUE TITLE
Double focus outline for checkbox and radio button when used with empty labels

### ISSUE THREAD

**Issue by InnaAtanasova** [SAP/fundamental-styles#1343]
When no label is provided (empty) the checkbox and the radio button get double focus outline

<img width="130" alt="Screen Shot 2020-08-11 at 12 27 42 PM" src="https://user-images.githubusercontent.com/39598672/89923157-2cd31b80-dbce-11ea-8249-d20df9020c8c.png">
<img width="133" alt="Screen Shot 2020-08-11 at 12 27 35 PM" src="https://user-images.githubusercontent.com/39598672/89923160-2d6bb200-dbce-11ea-9385-a8712ee6fd8d.png">
